litgpt/finetune/lora.py

```diff
@@ -262,10 +263,15 @@ def fit(
                     iter_num * train.micro_batch_size * model.config.block_size * fabric.world_size
                 ),
                 "learning_rate": scheduler.get_last_lr()[0],
+                "val_loss": val_loss,
             }
+            val_loss_str = 'None' if metrics['val_loss'] is None else f"{metrics['val_loss']:.3f}"
             fabric.print(
-                f"iter {metrics['iter']} | step {metrics['step']}: loss {metrics['loss']:.4f}, iter time:"
-                f" {metrics['iter_time'] * 1000:.2f} ms{' (optimizer.step)' if not is_accumulating else ''}"
+                f"Ep {metrics['epoch']+1} | iter {metrics['iter']} | step {metrics['step']}:"
```

Comment:
Maybe the full word Epoch is better (same in Lightning Trainer).
I agree with shortening optimizer.step